### PR TITLE
Support for material from hdf5 files

### PR DESCRIPTION
**Pull request by akva2**
As discussed in skype meeting etc. Downstream of https://github.com/OPM/IFEM/pull/395

### CODE REVIEW

File: LinIsotropic.C [hunk 1]
```diff
@@ -15,31 +15,34 @@
 #include "FiniteElement.h"
 #include "Field.h"
 #include "Functions.h"
-#include "Utilities.h"
+#include "FiniteElement.h"
```
Comment (kmokstad) [inline LinIsotropic.C]: FiniteElement.h and IFEM.h now included twice, please clean up

File: LinIsotropic.C [hunk 2]
```diff
@@ -189,6 +194,10 @@ bool LinIsotropic::evaluate (Matrix& C, SymmTensor& sigma, double& U,
   else if (Efunc)
     E = (*Efunc)(X);
 
+  double nu = nuC;
+  if (nuF)
+    nu = (*nuF)(X);
+
```
Comment (kmokstad) [inline LinIsotropic.C]: `double nu = nuF ? (*nuF)(X) : nuC;` like in l336

File: LinIsotropic.C [hunk 3]
```diff
@@ -289,6 +298,10 @@ bool LinIsotropic::evaluate (Matrix& C, SymmTensor& sigma, double& U,
 bool LinIsotropic::evaluate (double& lambda, double& mu,
                              const FiniteElement& fe, const Vec3& X) const
 {
+  double nu = nuC;
+  if (nuF)
+    nu = (*nuF)(X);
+
```
Comment (kmokstad) [inline LinIsotropic.C]: `double nu = nuF ? (*nuF)(X) : nuC;` like in l336

**Comment by VikingScientist** [comment]
I'd like to see this one revisited.

**Comment by kmokstad** [comment]
Closing this after over 6 months of inactivity. If I remember correctly the feature (material from HDF5) was no longer needed on the Elasticity level as it was handled by other apps.

Consider repoen/rebase later (2nd commit only) if the need should re-appear.

**Pull request closed without merging**